examples/llm/icl_eval/model_loading.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2022 MosaicML Examples authors
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Dict, Optional, Union
+
+import torch
+from composer.models.gpt2 import create_gpt2
+from composer.utils import reproducibility
+from omegaconf import OmegaConf as om
+from src.model_registry import COMPOSER_MODEL_REGISTRY
+from src.tokenizer import LLMTokenizer
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+
+def init_huggingface_causal_lm(
+        config: str) -> Dict[str, Union[AutoModelForCausalLM, AutoTokenizer]]:
+    return {
+        'model': create_gpt2(use_pretrained=True, pretrained_model_name=config)
+    }
+
+
+def init_composer_ckpt_from_yaml(
+    config: str, checkpoint: Optional[str]
+) -> Dict[str, Union[torch.nn.Module, LLMTokenizer, str]]:
+    """Load a MosaicGPT model and LLMTokenizer from a checkpoint.
```

Comment:
I don't think this docstring matches what this function does. I don't see any code that loads the checkpoint file, and a tokenizer is not part of the return type. Not sure if this is a docstring error, or the function is missing functionality, or I am misunderstanding something.

Reply:
yep I changed the function but not the docstring good catch!